Gemfile.jruby-1.9.lock

```diff
@@ -10,7 +10,7 @@ PATH
       i18n (= 0.6.9)
```

Comment:
@talevy I see no reason for that, anyway we will have to generate a new one for 2.3 and 2.2 releases.